### ISSUE TITLE
Give a low level design how we build a recommendation system for this application

### ISSUE THREAD

**Issue by Shu12388y** [Lets-code-with-us/scamwebsiteV1#30]
### ISSUE NAME

Give a low level design how we build a recommendation system for this application

### ISSUE DESCRIPTION

We have to build a simple recommendation system for the application. We need a low-level design of how we can build it. The framework and packages we need to build it.

### YOUR APPROACH

Give a brief explanation of how to built it in a doc form

### YOUR NAME

Shubham

### YOUR GITHUB PROFILE URL

https://github.com/shu12388y/

**Comment by gt951436** [comment]
Do we need to implement some design or just provide the approach in doc form?

**Comment by Shu12388y** [comment]
@gt951436 Yes steps on how we can build a recommendation system around it. How to build it using js or ts. Tech stack also

**Comment by gt951436** [comment]
@Shu12388y assign it to me

**Comment by gt951436** [comment]
file format should strictly be in doc form or .md file will be good?

**Comment by Shu12388y** [comment]
doc will be good you can also add the diagrams

**Comment by Shu12388y** [comment]
@gt951436 Any updates?
